daffodil-core/src/main/scala/org/apache/daffodil/core/runtime1/TermRuntime1Mixin.scala

```diff
@@ -354,8 +355,12 @@ trait TermRuntime1Mixin { self: Term =>
           case (Closed(pnes1), Closed(pnes2)) => {
             thisItself // When everything is closed, we only want thisItself and not the entire list of Closed possibilities
           }
-          case (Open(pnes1), Closed(pnes2)) => {
+          case (Open(pnes1), Closed(pnes2)) if !this.isInstanceOf[ChoiceGroupRef] => {
```

Comment:
I can seen an argument to be made here that this logic should actually be part of SeveralPossibilitiesForNextElement in PartialNextElementResolver for when we are determining if the element is actually required or not.  Figured I'd leave it up for discussion.

Reply:
I guess one benefit here is that this is checked at compile time instead of runtime? Seems like the preferred logic.

Reply:
This seems ok to me here.